From e65fe99b030e8955f522be7202a2e76f39cc9645 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ralf=20Cors=C3=A9pius?= <corsepiu@fedoraproject.org>
Date: Tue, 18 Mar 2014 14:47:19 +0100
Subject: [PATCH 3/8] Broken test-dependencies.

---
 sbin/rt-test-dependencies.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sbin/rt-test-dependencies.in b/sbin/rt-test-dependencies.in
index f8b80b7..b7d19d8 100644
--- a/sbin/rt-test-dependencies.in
+++ b/sbin/rt-test-dependencies.in
@@ -189,7 +189,7 @@ CGI 3.38
 CGI::Cookie 1.20
 CGI::Emulate::PSGI
 CGI::PSGI 0.12
-Class::Accessor 0.34
+Class::Accessor::Fast 0.34
 Crypt::Eksblowfish
 CSS::Squish 0.06
 Data::GUID
-- 
2.1.0

