From 26d66993512c6beb90eedb91c6d3c54c8bd1d953 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ralf=20Cors=C3=A9pius?= <corsepiu@fedoraproject.org>
Date: Tue, 18 Mar 2014 14:59:31 +0100
Subject: [PATCH 3/5] Remove fixperms, font-install.

---
 Makefile.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 276b27f00..37a88dacb 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -267,7 +267,7 @@ upgrade-instruct:
 	@echo "    make upgrade-database"
 
 
-upgrade: testdeps config-install dirs files-install fixperms upgrade-instruct
+upgrade: testdeps config-install dirs files-install upgrade-instruct
 
 testdeps:
 	$(PERL) ./sbin/rt-test-dependencies
@@ -363,9 +363,9 @@ clean-mason-cache:
 	rm -rf $(DESTDIR)$(MASON_DATA_PATH)/etc/*
 	rm -rf $(DESTDIR)$(MASON_DATA_PATH)/obj/*
 
-install: testdeps config-install dirs files-install fixperms instruct
+install: testdeps config-install dirs files-install instruct
 
-files-install: libs-install etc-install config-install bin-install sbin-install html-install doc-install font-install po-install static-install
+files-install: libs-install etc-install config-install bin-install sbin-install html-install doc-install po-install static-install
 
 config-install:
 @COMMENT_INPLACE_LAYOUT@	$(INSTALL) -m 0755 -o $(BIN_OWNER) -g $(RTGROUP) -d $(DESTDIR)$(CONFIG_FILE_PATH)
-- 
2.31.1

