From ccc9009326df42978db11ba23b07d375270cab8a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ralf=20Cors=C3=A9pius?= <corsepiu@fedoraproject.org>
Date: Tue, 18 Jan 2022 09:41:09 +0100
Subject: [PATCH 6/6] Add fcgid

---
 sbin/rt-test-dependencies.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sbin/rt-test-dependencies.in b/sbin/rt-test-dependencies.in
index 99f407b68..ec1fe2703 100644
--- a/sbin/rt-test-dependencies.in
+++ b/sbin/rt-test-dependencies.in
@@ -62,7 +62,7 @@ GetOptions(
     \%args,
     'install!',
     'with-MYSQL', 'with-PG', 'with-SQLITE', 'with-ORACLE',
-    'with-FASTCGI', 'with-MODPERL2', 'with-STANDALONE',
+    'with-FASTCGI', 'with-MODPERL2', 'with-FCGID', 'with-STANDALONE',
 
     'with-DEVELOPER',
 
@@ -102,7 +102,7 @@ my %default = (
 );
 
 $default{"with-".uc("@DB_TYPE@")} = 1 unless grep {$args{"with-$_"}} qw/MYSQL PG SQLITE ORACLE/;
-unless (grep {$args{"with-$_"}} qw/FASTCGI MODPERL1 MODPERL2 STANDALONE/) {
+unless (grep {$args{"with-$_"}} qw/FASTCGI MODPERL1 MODPERL2 FCGID STANDALONE/) {
     $default{"with-".uc($_)} = 1 for grep {defined && length} split /,/, "@WEB_HANDLER@"
 }
 
-- 
2.37.1

