diff --git a/lib/Log/Dispatch/Syslog.pm b/lib/Log/Dispatch/Syslog.pm
index 0af2cd3..cb9cb3e 100644
--- a/lib/Log/Dispatch/Syslog.pm
+++ b/lib/Log/Dispatch/Syslog.pm
@@ -157,7 +157,7 @@ Defaults to ''.
 
 Specifies what type of program is doing the logging to the system log.
 Valid options are 'auth', 'authpriv', 'cron', 'daemon', 'kern',
-'local0' through 'local7', 'mail, 'news', 'syslog', 'user',
+'local0' through 'local7', 'mail', 'news', 'syslog', 'user',
 'uucp'.  Defaults to 'user'
 
 =item * socket ($ or \@)
diff --git a/t/release-pod-spell.t b/t/release-pod-spell.t
index b480f66..18235a1 100644
--- a/t/release-pod-spell.t
+++ b/t/release-pod-spell.t
@@ -27,11 +27,13 @@ local $ENV{LC_ALL} = 'C';
 all_pod_files_spelling_ok;
 
 __DATA__
+''
 API
 CPAN
 Cholet
 Dumont
 Goess
+Log4perl
 Manfredi
 Miyagawa
 PayPal
@@ -46,21 +48,34 @@ Swartz
 Tatsuhiko
 UTF
 apache
+'append'
 appenders
-auth
-authpriv
+'auth'
+'authpriv'
 autoflushed
 classname
-crit
-emerg
+'crit'
+'daemon'
+'emerg'
+'err'
 filename
 ident
-kern
+'kern'
+'local0'
+'local7'
+log4j
 logopt
+'mail'
+'message'
 multi
+'news'
 params
 smtp
 stderr
 subclass's
 subclasses
-uucp
+'user'
+'uucp'
+'warn'
+'warning'
+'write'
